packages/frontend/src/redux/slices/ledger/index.js

```diff
@@ -16,14 +18,73 @@ const SLICE_NAME = 'ledger';
 export const LEDGER_MODAL_STATUS = {
     CONFIRM_PUBLIC_KEY: 'confirm-public-key',
     CONFIRM_ACCOUNTS: 'confirm-accounts',
-    ENTER_ACCOUNTID: 'enter-accountId'
+    ENTER_ACCOUNTID: 'enter-accountId',
+    SUCCESS: 'success'
+};
+
+export const CONNECT_MODAL_TYPE = {
+    CONNECT: 'connect',
+    CONNECTION_ERROR: 'connection-error',
+    DISCONNECTED: 'disconnected',
+    HIDE: false
```

Comment:
Can we make this a string for the sake of consistency? If we don't need to rely on it being a bool then it should have the same type as the other constants. I think it would also be more clear when querying the modal's `type` to get `"hidden"` rather than just `false`

Reply:
Your right, it looks odd :) I just removed it, and now we are setting `type` to `undefined` if we don't want to show modal. Is that ok?